src/routes/FilterInterface.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
```

Comment:
there is still some typescript... remove it and us JSDoc instead.
```suggestion
<script>
```